tests/fail/shims/memrchr_null.stderr

```diff
@@ -1,15 +1,28 @@
-error: Undefined Behavior: memory access failed: null pointer is a dangling pointer (it has no provenance)
+error[E0425]: cannot find function `memrchr` in crate `libc`
```

Comment:
Looks like the blessed output now says "cannot find function"? That doesn't seem right?^^